CONTRIBUTING.md

```diff
@@ -312,6 +314,30 @@ $ yarn watch & # runs in the background
 * Make sure to update the PR title/description if things change. The PR title/description are going to be used as the
   commit title/message and will appear in the CHANGELOG, so maintain them all the way throughout the process.
 
+#### Adding new dependencies
```

Comment:
Is this section about adding (all) new dependencies or just "unconventional" ones?

Reply:
Just "unconventional" ones. Updated the title to reflect this.